sqlmesh/core/console.py

```diff
@@ -565,8 +590,15 @@ def _prompt_backfill(self, plan: Plan, auto_apply: bool) -> None:
                 )
                 if end:
                     plan.end = end
-
-        if not auto_apply and self._confirm(f"Apply - {backfill_or_preview.capitalize()} Tables"):
+        if plan.ignored_snapshot_names:
+            self._print(
+                self._get_ignored_tree(plan.ignored_snapshot_names, plan.context_diff.snapshots)
+            )
+        if (
+            plan.has_changes
```

Comment:
Why did you have to add this? Don't you also need `plan.has_unmodified_unpromoted`?

Reply:
You are right that this is not needed. Thanks for catching.